app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1>User profile</h1>
\ No newline at end of file
+<div>
+<%= render partial: "render", users: @users%>
+<div class="bio-container">
+<h2>Bio</h2>
+<p><%= @users[0].bio %></p>
+</div>
+
+<% @posts.each do |post| %>
```

Comment:
- [ ] Great job working on this milestone. As per project requirements and as per the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAACH6LHD4CS32SZ7FY4I5GP54ZI52MMQ) you are supposed to render the three most recent posts. To achieve this, you are to make use of the `recent_posts` method in your user model.
`<% @user.recent_posts.each do |post| %>`